css/src/alerts.css

```diff
@@ -0,0 +1,37 @@
+.alert {
+	display: flex;
+	align-items: baseline;
+	position: relative;
```

Comment:
Pre-existing: not sure `position: relative;` does anything.

Reply:
True, the `position: relative;` isn't needed. As is the `font-weight: 400;` because the standard font-weight is always 400 unless stated otherwise.